etl/steps/data/garden/state_capacity/2023-10-19/state_capacity_dataset.meta.yml

```diff
@@ -0,0 +1,447 @@
+          name: Efficiency of Revenue Mobilization
+          numDecimalPlaces: 1
+      milexpercap:
+        title: (log) Military expenditures per capita (SIPRI/COW)
+        unit: ""
+        short_unit: ""
+        description_short: Measures the amount of money spent on the military per capita. It is logged to reduce the skewness of the distribution.
+        description_from_producer: >-
+          Data on military spending from the SIPRI Military Expenditure Database (2020), which cover
+          the years 1949-2019. Missing data are filled from from the Correlates of War Project’s National
+          Military Capabilities dataset version 5.0 (Singer et al. 1972; Singer 1987). Population data are
+          from the United Nations World Population Prospects (2019).
+        processing_level: minor
+        display:
+          name: (log) Military expenditures per capita
+          numDecimalPlaces: 1
+      milpercap:
+        title: (log) Military personnel per 1,000 people (SIPRI/COW)
+        unit: ""
+        short_unit: ""
+        description_short: Measures the number of military personnel per 1,000 people. It is logged to reduce the skewness of the distribution.
+        description_from_producer: >-
+          Data on the number of military personnel (in thousands) from 1960-2012 comes primarily from
+          the Correlates of War Project’s National Military Capabilities dataset version 5.0 (Singer et al.
+          1972; Singer 1987). Data on military personnel for 2013-2015 come from the World Development
+          Indicators (World Bank). Population data are from the United Nations World Population Prospects
+          (2019).
+        processing_level: minor
+        display:
+          name: (log) Military personnel per 1,000 people
+          numDecimalPlaces: 1
+      policecap:
+        title: (log) Police Officers per 1,000 people (UN)
+        unit: ""
+        short_unit: ""
+        description_short: Measures the number of police officers per 1,000 people. It is logged to reduce the skewness of the distribution.
+        description_from_producer: >-
+          Data on police personnel come from the United Nations Survey of Crime Trends and Operations of
+          Criminal Justice Systems (UNODC) from various years. These are country-reported data covering
+          the years 1973-2015 for a varying number of countries each year.
+        processing_level: minor
+        display:
+          name: (log) Police Officers per 1,000 people
+          numDecimalPlaces: 1
+      statehist50s:
+        title: State Antiquity Index (50 year discounting)
+        unit: ""
+        short_unit: ""
+        description_short: Measures the antiquity and sovereignty of the state. It ranges from 0 to 1, with higher values indicating greater antiquity and sovereignty.
+        description_from_producer: >-
+          In the original, this index codes three items for contemporary countries, based on 1950 borders,
```

Comment:
"In the original" -> "In the original data"?

Reply:
Yeah, it's the way it is from the source. I will change it.